README.md

```diff
@@ -2,20 +2,104 @@
 
 > Test client for Xo-Server
 
-## Usage
+You should have a `admin@admin.net` account with password `admin`.
 
-xo-server needs to be installed on `localhost:9000`.
-You should also have a `admin@admin.net` account with password `admin`.
+## Adding a test
 
-There should be a Xen Server available at `192.168.1.3`with the credentials `root` and password `qwerty`.
+### Organization
 
-Warning ! This client will create and delete a lot of items (users, groups, servers ...) to test features
+```
+src
+├─ user
+|   ├─ __snapshots__
+|   |     └─ index.spec.js.snap 
+|   └─ index.spec.js
+├─ job
+¦   └─ index.spec.js
+¦
+¦
+├─ _xoWithTestHelpers.js
+└─ util.js
+```
 
+The tests can describe xo methods or scenarios:  
 ```
-> npm ci
-> npm test
+describe("user", () => {
+  // testing a method
+  describe(".create(): ", () => {
+    it("creates a user with permission", () => {
+      /* some tests */
+    });
+  });
+      
+  // testing a scenario
+  if("creates two users, modifies a user email to be the same with the other and fails trying to connect them", () => {
```

Comment:
I'm not sure that's `user.create()` does all that ;-)

Reply:
It's a scenario. At the end of the `describe("user", () => ... )`, there will be all the scenarios about users, not only about the method `user.create()`.

Reply:
I'm not sure using `it()` is the best to describe a scenario, because of the phrase it constructs: `user.create() creates two users, modifies a user email to be the same with the other and fails trying to connect them`.

Reply:
The result will be: `user creates two users, modifies a user email to be the same with the other and fails trying to connect them`. The `it()` is outside `describe("create()", () => ... )` .
Do you mean that we have to change the sentence structure of the result of the tests? Or, maybe just use `test()` instead of `it()` ?

Reply:
Ok, the structure is fine then.

But I don't think it makes sense semantically to use `it()` outside of a `describe()`, for a scenario, I would use `test()` instead.